src/main/java/org/jitsi/jigasi/transcription/LocalTxtTranscriptHandler.java

```diff
@@ -55,6 +56,18 @@
      */
     private static final String NEW_LINE = System.lineSeparator();
 
+    /**
+     * Property name for getting a target language for translation.
+     */
+    public final static String P_NAME_TARGET_LANGUAGE
+            = "org.jitsi.jigasi.transcription.TARGET_LANGUAGE";
+
+    /**
+     * Whether to translate text before sending results and the target language.
+     */
+    public final static String TARGET_LANGUAGE
```

Comment:
What do you think about renaming to "DEFAULT_TARGET_LANGUAGE"? I think it's more descriptive

Reply:
Or you could remove it and just use ""